relayer/src/worker/client.rs

```diff
@@ -14,6 +14,9 @@ use crate::{
 
 use super::WorkerCmd;
 
+const TRUSTING_PERIOD_DIVIDER: u32 = 100;
```

Comment:
It seems like for Tendermint clients we would refresh every ~3 hours if the trusting period is 18 days. This seems a bit too infrequest. We should retry more frequently.